src/Document/Library/Bridge/Symfony/DependencyInjection/ZenstruckDocumentLibraryExtension.php

```diff
@@ -90,6 +94,20 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
             ->addTag('form.type')
         ;
 
+        // value resolver
+        $container->register('.zenstruck_document.value_resolver.extractor', RequestFilesExtractor::class)
+            ->addArgument(
+                (new Definition(PropertyAccessor::class))->setArguments([
+                    PropertyAccessor::DISALLOW_MAGIC_METHODS,
+                    PropertyAccessor::THROW_ON_INVALID_PROPERTY_PATH,
+                ])
+            )
+        ;
```

Comment:
```suggestion
            ->addArgument(new Reference('property_accessor'))
        ;
```

Until a time where the default `property_accessor` causes a problem.